src/elm/Page/Dashboard/Balance.elm

```diff
@@ -1,24 +1,21 @@
-module Page.Dashboard.Community exposing (Model, Msg, UpdateResult, init, jsAddressToMsg, msgToString, update, viewCard, viewTableRow)
```

Comment:
Hmmm I cam confused how come we are naming this module `Balance` it looks like it is doing more than handling balances ....

Reply:
before that it was called “community” I’ve changed it because it was actually the individual balance card we see on the dashboard. The other reason is that it was very hard to find by querying the project files since we have a bunch of different “community” files around 

i wouldn’t worry too much about it, the new dashboard will remove this file entirely